authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+from .models import Profile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    username = serializers.SerializerMethodField()
+
+    first_name = serializers.CharField(allow_blank=True, required=False,
+                                       min_length=2, max_length=50)
+    last_name = serializers.CharField(allow_blank=True, required=False,
+                                      min_length=2, max_length=50)
+    bio = serializers.CharField(allow_blank=True, required=False)
+
+    image = serializers.URLField(allow_blank=True, required=False)
+
+
+    class Meta:
+        model = Profile
+        fields = ('username','first_name','last_name','created_at','bio','following','date_of_birth','image')
+    @classmethod
+    def get_username(self, obj):
```

Comment:
@Rhytah , why are you adding the username field on to this serializer yet on signup the user provide there username and there is also an endpoint for updating the username.